pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -647,15 +639,17 @@ func (a *ofLearnAction) LoadXXRegToXXReg(fromXXField, toXXField *XXRegField) Lea
 	return a
 }
 
-func (a *ofLearnAction) LoadRegMark(mark *RegMark) LearnAction {
-	toField := &ofctrl.LearnField{Name: mark.field.GetNXFieldName(), Start: uint16(mark.field.rng[0])}
-	valBuf := make([]byte, 4)
-	binary.BigEndian.PutUint32(valBuf, mark.value)
-	offset := (mark.field.rng.Length()-1)/8 + 1
-	if offset < 2 {
-		offset = 2
+func (a *ofLearnAction) LoadRegMark(marks ...*RegMark) LearnAction {
+	for _, mark := range marks {
+		toField := &ofctrl.LearnField{Name: mark.field.GetNXFieldName(), Start: uint16(mark.field.rng[0])}
+		valBuf := make([]byte, 4)
+		binary.BigEndian.PutUint32(valBuf, mark.value)
+		offset := (mark.field.rng.Length()-1)/8 + 1
```

Comment:
```suggestion
		offset := (mark.field.rng.Length()+7)/8
```

Reply:
Looks more readable.

Reply:
Will you update it? Currently the code is different in MatchRegMark and LoadRegMark

Reply:
I'll update them to the same.